core/debug/src/debug.c

```diff
@@ -60,83 +60,6 @@ void debug_init(void)
     SCnSCB->ACTLR |= 0x2;
 }
 
-static void debug_cx_state(int _indent)
```

Comment:
The reason I didn't do it is that this requites slightly more change than the rest of the code, because it uses some internal states that are not exposed. The function is very old anyway and it was not showing all meaningful information (I was planning to change it anyway).

If it's ok I'd still remove it and add a ticket to re-implement context-switch debugging. Currently debug for context switches was disabled anyway.